CHANGES/README.rst

```diff
@@ -1,15 +1,23 @@
-.. _Adding change notes with your PRs:
```

Comment:
Changing this label is going to be problematic as other projects rely on it for referring the changelog guidelines from other places. Not all the PR recommendations set. It'll have to be reverted.